packages/components-css/src/index.tsx

```diff
@@ -1,12 +1,10 @@
 import '../../../components/Alert/src/index.scss';
 import '../../../components/Accordion/src/index.scss';
 import '../../../components/AnchorNavigation/src/index.scss';
-import '../../../components/AuthenticationBlock/src/index.scss';
```

Comment:
Why are you removing blocks which area already on the `main` branch?

Reply:
fixt it